jest.config.js

```diff
@@ -1,5 +1,204 @@
+  // rootDir: undefined,
+
+  // A list of paths to directories that Jest should use to search for files in
+  // roots: [
+  //   "<rootDir>"
+  // ],
+
+  // Allows you to use a custom runner instead of Jest's default test runner
+  // runner: "jest-runner",
+
+  // The paths to modules that run some code to configure or set up the testing environment before each test
+  // setupFiles: [],
+
+  // A list of paths to modules that run some code to configure or set up the testing framework before each test
+  // setupFilesAfterEnv: [],
+
+  // The number of seconds after which a test is considered as slow and reported as such in the results.
+  // slowTestThreshold: 5,
+
+  // A list of paths to snapshot serializer modules Jest should use for snapshot testing
+  // snapshotSerializers: [],
+
+  // The test environment that will be used for testing
+  testEnvironment: 'jsdom',
+
+  // Options that will be passed to the testEnvironment
+  // testEnvironmentOptions: {},
+
+  // Adds a location field to test results
+  // testLocationInResults: false,
+
+  // The glob patterns Jest uses to detect test files
+  // testMatch: [
+  //   "**/__tests__/**/*.[jt]s?(x)",
+  //   "**/?(*.)+(spec|test).[tj]s?(x)"
+  // ],
+
+  // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
+  // testPathIgnorePatterns: [
+  //   "/node_modules/"
+  // ],
+
+  // The regexp pattern or array of patterns that Jest uses to detect test files
+  // testRegex: [],
+
+  // This option allows the use of a custom results processor
+  // testResultsProcessor: undefined,
+
+  // This option allows use of a custom test runner
+  // testRunner: "jest-circus/runner",
+
+  // A map from regular expressions to paths to transformers
+  // transform: undefined,
+
+  // An array of regexp pattern strings that are matched against all source file paths, matched files will skip transformation
+  // transformIgnorePatterns: ['/node_modules/', '\\.pnp\\.[^\\/]+$']
+
+  transformIgnorePatterns: [
+    'node_modules/(?!(react-native' +
+      '|react-navigation-tabs' +
+      '|react-native-splash-screen' +
+      '|react-native-screens' +
+      '|react-native-reanimated' +
+      ')/)',
+    'node_modules/(?!(jest-)?react-native|react-(native|universal|navigation)-(.*)|@react-native-community/(.*)|@react-navigation/(.*)|bs-platform|(@[a-zA-Z]+/)?(bs|reason|rescript)-(.*)+)',
+    '/node_modules/'
+  ]
+
+  // An array of regexp pattern strings that are matched against all modules before the module loader will automatically return a mock for them
+  // unmockedModulePathPatterns: undefined,
+
+  // Indicates whether each individual test should be reported during the run
+  // verbose: undefined,
+
+  // An array of regexp patterns that are matched against all source file paths before re-running tests in watch mode
+  // watchPathIgnorePatterns: [],
+
+  // Whether to use watchman for file crawling
+  // watchman: true,
```

Comment:
Due to the issue in https://github.com/LiorKGOW/My-Guitar-Store/actions/runs/3486587845/jobs/5833234780#step:4:65

can you add the following? it will ignore that `scss` files.
```js
moduleNameMapper: {
    '^.+\\.(css|scss)$': 'identity-obj-proxy',
}
```

Reply:
you will need to add `identity-obj-proxy` to the `package.json` file

Reply:
I'll add it 
Thanks @Ron-Lavi

Reply:
> you will need to add `identity-obj-proxy` to the `package.json` file

The values there are dictionary
what should be it's value? @Ron-Lavi 

I am getting this error message: 
yarn run v1.22.10
$ jest
 PASS  test/javascript/example2.test.js
 PASS  test/javascript/example1.test.js
 FAIL  test/javascript/GuitarGallery.test.js
  ● Test suite failed to run

    Configuration error:
    
    Could not locate module ./backdrop.css mapped as:
    identity-obj-proxy.
    
    Please check your configuration for these entries:
    {
      "moduleNameMapper": {
        "/^.+\.(css|scss)$/": "identity-obj-proxy"
      },
      "resolver": undefined
    }

      1 | import React, { useEffect, useState } from 'react';
      2 | import axios from 'axios';
    > 3 | import { Title, TitleSizes, Gallery } from '@patternfly/react-core';
        | ^
      4 | import GuitarInfo from './GuitarInfo';
      5 | import guitarsUrl from '../constUrls';
      6 | import styled from 'styled-components';

      at createNoMappedModuleFoundError (node_modules/jest-resolve/build/resolver.js:752:17)
      at Object.<anonymous> (node_modules/@patternfly/react-styles/css/components/Backdrop/backdrop.js:3:1)
      at Object.<anonymous> (node_modules/@patternfly/react-core/src/components/AboutModal/AboutModal.tsx:4:1)
      at Object.<anonymous> (node_modules/@patternfly/react-core/src/components/AboutModal/index.ts:1:1)
      at Object.<anonymous> (node_modules/@patternfly/react-core/src/components/index.ts:2:1)
      at Object.<anonymous> (node_modules/@patternfly/react-core/src/index.ts:1:1)
      at Object.<anonymous> (app/javascript/components/GuitarGallery.js:3:1)
      at Object.<anonymous> (test/javascript/GuitarGallery.test.js:5:1)

 FAIL  config/webpack/test.js
  ● Test suite failed to run

    Configuration error:
    
    Could not locate module ./module.css mapped as:
    identity-obj-proxy.
    
    Please check your configuration for these entries:
    {
      "moduleNameMapper": {
        "/^.+\.(css|scss)$/": "identity-obj-proxy"
      },
      "resolver": undefined
    }

    > 1 | const { environment } = require('@rails/webpacker');
        |                         ^
      2 | const { merge } = require('webpack-merge');
      3 | const customConfig = require('./custom');
      4 |

      at createNoMappedModuleFoundError (node_modules/jest-resolve/build/resolver.js:752:17)
      at Object.<anonymous> (node_modules/@rails/webpacker/package/rules/index.js:5:19)
      at Object.<anonymous> (node_modules/@rails/webpacker/package/environments/base.js:20:15)
      at Object.<anonymous> (node_modules/@rails/webpacker/package/index.js:6:21)
      at Object.require (config/webpack/environment.js:1:25)
      at Object.require (config/webpack/test.js:3:21)

Test Suites: 2 failed, 2 passed, 4 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        5.861 s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.